app/controllers/lightweight_activities_controller.rb

```diff
@@ -122,16 +122,6 @@ def print_blank
     setup_show
   end
 
-  def summary
```

Comment:
It is possible there will be some old summary links out there that students sent to teachers. So it would be better if this action would redirect to the new portal-report links.